src/views/forms/TradeForm/AdvancedTradeOptions.tsx

```diff
@@ -144,8 +159,12 @@ export const AdvancedTradeOptions = () => {
             )}
             {showReduceOnly && (
               <Checkbox
-                checked={(reduceOnly && !reduceOnlyTooltip) || false}
-                disabled={!!reduceOnlyTooltip}
+                checked={
+                  (reduceOnly && !reduceOnlyTooltip) ||
+                  complianceState === ComplianceStates.CLOSE_ONLY ||
+                  false
+                }
+                disabled={!!reduceOnlyTooltip || complianceState === ComplianceStates.CLOSE_ONLY}
```

Comment:
nit: I think it's mildly confusing to have something checked and disabled. Can we add a tooltip explaining why in the close only state?

Reply:
Oh great catch. I missed it in the design. there is actually an error message showing the warning to the user. Added it now.
<img width="320" alt="Screenshot 2024-04-24 at 10 05 43 PM" src="https://github.com/dydxprotocol/v4-web/assets/1062252/76ca79e1-d55f-408f-bb4d-e8dbb11fac3d">